api/comments/serializers.py

```diff
@@ -111,6 +113,8 @@ def update(self, comment, validated_data):
                     comment.edit(content, auth=auth, save=True)
                 except PermissionsError:
                     raise PermissionDenied('Not authorized to edit this comment.')
+                except ValidationValueError:
+                    raise ValidationError('Ensure this field has no more than 500 characters.')
```

Comment:
Oh.. I think I understand how the `comment_maxvalidator` is actually validating against the number of input characters. So no need to be too generic with the error message. I suppose you could use the value of `settings.COMMENT_MAXLENGTH` though.